Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -997,12 +1002,20 @@ extension ChatViewController {
 
     // MARK: Cells
 
+    func cellForEmpty(at indexPath: IndexPath) -> UICollectionViewCell {
+        if let cell = collectionView?.dequeueReusableCell(withReuseIdentifier: kEmptyCellIdentifier, for: indexPath) {
+            return cell
+        }
+
+        return UICollectionViewCell()
+    }
+
     func cellForMessage(_ obj: ChatData, at indexPath: IndexPath) -> UICollectionViewCell {
         guard let cell = collectionView?.dequeueReusableCell(
             withReuseIdentifier: ChatMessageCell.identifier,
             for: indexPath
         ) as? ChatMessageCell else {
-            return UICollectionViewCell()
+            return cellForEmpty(at: indexPath)
```

Comment:
Does this fix a crash tracked in fabric.io?

Reply:
Yes, this one: https://fabric.io/rocketchat3/ios/apps/chat.rocket.ios/issues/5ad57a2836c7b23527fbbcb1?time=last-seven-days